package-lock.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "radar-sdk-js",
-  "version": "3.0.0-beta.8",
+  "version": "3.0.1",
```

Comment:
does this actually need to be bumped? or only with a fresh `npm i`? doesn't seem like we ever did it for v3